awesome_cart/public/css/awc_cart.css

```diff
@@ -571,11 +571,11 @@ body.is_logged_in .coupon_input.no-login {
 }
 
 input:checked + .slide {
-	background-color: #000000;/*#2196F3;*/
+	background-color: #2196F3;
 }
 
 input:focus + .slide {
-	box-shadow: 0 0 1px #000000;/*#2196F3;*/
+	box-shadow: 0 0 1px#2196F3;
```

Comment:
Does this work without the space?

Reply:
Yeah it works on due to the # boundary but its bad form. Fixed it on current commit